compiler/src/org.graalvm.compiler.core.aarch64.test/src/org/graalvm/compiler/core/aarch64/test/AArch64AddSubShiftTest.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, Arm Limited and affiliates. All rights reserved.
```

Comment:
Our internal verification points out, that this copyright header is incorrect. At the moment I am waiting for input what we should do about it. I'll follow up on this later.

Reply:
Thanks for your reviewing! And waiting for your correct copyright!

Reply:
Hi sanzinger,
Is the correct copyright ok? Thanks！